opm/simulators/wells/FractionCalculator.hpp

```diff
@@ -56,10 +56,12 @@ class FractionCalculator
 
 private:
     std::string parent(const std::string& name);
-    Scalar guideRateSum(const Group& group,
-                        const std::string& always_included_child);
+    std::pair<Scalar,int> guideRateSum(const Group& group,
```

Comment:
Add docs here and for guideRate(). The new return value and parameter must be documented at least.

Nitpick: space after comma.